### DISCUSSION TITLE
Change database data

### DISCUSSION THREAD

**Discussion by mhdaouas** [directus/directus#14770] [Archive]
I want to change a collection item as soon as I create it based on a flow. I tried creating a flow with 2 connected nodes. A creation event hook tied to a transform payload node. However after I created an item, the data didn't change (I changed the status).

**Comment by mhdaouas** [comment]
Up :(

**Reply by rijkvanzanten** [reply]
No need to up it, saw your question 😉 Haven't had the time to recreate the setup though to investigate what might be going on and how to set this up. [Discord](https://directus.chat) might be a faster way to get answers to usage questions like these, as there's a larger user base of developers reading along 🙂

**Reply by mhdaouas** [reply]
I couldn't find an answer to my question. I need to change an item data with the transform payload as soon as I create it but it doesn't work.

**Reply by rijkvanzanten** [reply]
@nickrum thoughts?

**Reply by nickrum** [reply]
By default the Response Body option of the Filter trigger isn't set to any key, which means the payload will be forwarded as is. To change the payload, it needs to be set to the key of the operation that the new payload should come from.
